native/exe/native

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/ruby -wU
+#!/usr/bin/ruby -wUI/opt/driver/src/build/dependencies
```

Comment:
This won't work locally, right?

Reply:
Mmmm, I don't remember why I did that, but both `make test` and rebuilding the docker image work without it, I'll remove it and see if CI works here.